### PR TITLE
add a note according CONTENT IDEA REQUEST 100702

### PR DESCRIPTION
**Pull request by TobyTu**

### CODE REVIEW

**Comment by TobyTu** [comment]
#sign-off

**Comment by TobyTu** [comment]
@jambirk,  CrystalThomasMS, kindly have a review and help merge the pull request.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/network-requirements/network-requirements.md [hunk 1]
```diff
@@ -124,6 +124,10 @@ The media traffic bandwidth usage can be challenging to calculate because of the
 |SILK Wideband  <br/> |Peer-to-peer  <br/> |20.0  <br/> |36.0  <br/> |48.0  <br/> |68.0  <br/> |
 |SILK wideband/narrowband  <br/> |Peer-to-peer  <br/> |13.0  <br/> |29.0  <br/> |41.0  <br/> |54.0  <br/> |
 
+> [!NOTE]
+> If enough bandwidth is not available for that codec, then calls can fail with an error that resembles the following in the Media logs. Note: Media logs (.blog) files can be decoded by Microsoft support personnel only.
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/network-requirements/network-requirements.md]:
```suggestion
> If there is not enough bandwidth available for that codec, then calls can fail with an error in the Media logs that resembles the following: “Atleast one codec must be enabled, hr: c0042004”.  Note that Media logs (.blog) files can only be decoded by Microsoft support personnel.
```

File: Skype/SfbServer/plan-your-deployment/network-requirements/network-requirements.md [hunk 2]
```diff
@@ -124,6 +124,10 @@ The media traffic bandwidth usage can be challenging to calculate because of the
 |SILK Wideband  <br/> |Peer-to-peer  <br/> |20.0  <br/> |36.0  <br/> |48.0  <br/> |68.0  <br/> |
 |SILK wideband/narrowband  <br/> |Peer-to-peer  <br/> |13.0  <br/> |29.0  <br/> |41.0  <br/> |54.0  <br/> |
 
+> [!NOTE]
+> If enough bandwidth is not available for that codec, then calls can fail with an error that resembles the following in the Media logs. Note: Media logs (.blog) files can be decoded by Microsoft support personnel only. 
+“Atleast one codec must be enabled, hr: c0042004”.
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/network-requirements/network-requirements.md]:
```suggestion
```

**Comment by TobyTu** [comment]
Meera will work on with the writer to update this article directly, close this PR

**Pull request closed without merging**